jobs/ml-ops/terraform/variables.tf

```diff
@@ -25,15 +25,17 @@ variable "s3_url" {
   default = "https://s3.fr-par.scw.cloud"
 }
 
-variable "data_file" {
-  type        = string
-  description = "name data file in data store"
-  default = "bank_telemarketing.csv"
+variable "data_fetch_cron_schedule" {
+  type = string
+  default = "* */10 * * *"
```

Comment:
**issue(broken)**: this will run every minute past every 10th hour, i.e. at 10:00, 10:01, 10:02

Please check all of these CRONs with https://crontab.guru and make sure they are what you expect.

Reply:
Yes, it's `0 */10 * * *`. Fixed all CRON schedules.